src/Microsoft.Diagnostics.EventFlow/Microsoft.Diagnostics.EventFlow.csproj

```diff
@@ -0,0 +1,11 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard1.6;netstandard2.0;net462;net471</TargetFrameworks>
```

Comment:
Do we need all of these target frameworks? Or can we get away with, for example, only specifying `netstandard1.6` here?

Reply:
In the class `MemberInfoConverter` there are conditional compilation directives. 

For two cases of `NETSTANDARD1_6` and `NET462 || NET471 || NETSTANDARD2_0` there are differences in the definition of the class and the methods that are overridden. For example, in the former case `MemberInfoConverter` derives from `JsonConverter`, but in the latter - from `JsonConverter<MemberInfo>`.

I am not sure what was the purpose of this approach, so I think it is more safe to leave `TargetFrameworks` and `MemberInfoConverter` as they are.

Reply:
Sounds good. I was just wondering.